### ISSUE TITLE
Feature Request: Uploading/Downloading in parallel chunks

### ISSUE THREAD

**Issue by SvenAbels** [abraunegg/onedrive#2068]
### Is your feature request related to a problem? Please describe.

The OneDrive client for Mac and Windows seem to upload / download 3 files simultaneously. This seems to speed up the process a lot for smaller files as there seems to be a small delay between each upload. For example, the following shows the network usage when syncing multiple files at the moment:

![image](https://user-images.githubusercontent.com/1993969/182156313-ff253fe2-87fe-4d29-966e-9e7813105d6b.png)

### Describe the solution you'd like

see above

### Describe alternatives you've considered

see above

**Comment by abraunegg** [comment]
@SvenAbels 
Closing this as duplicate of https://github.com/abraunegg/onedrive/issues/232

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
